web/projectsEvaluation.php

```diff
@@ -394,13 +394,41 @@ function loadProjects() {
             text: 'Extended view',
             handler: showExtendedView,
         }],
+        initComponent: function() {
+            this.tbar = this.buildTopToolbar();
+            Ext.ux.ExportableGridPanel.superclass.initComponent.apply(this, arguments);
+        },
```

Comment:
This method is only required to call `buildTopToolbar()` inside the context of the `projectGrid` object, so `this` refers to it. Let's do things more simple by removing the entire `initComponent` method and doing things a bit differently.